actions/class.Runner.php

```diff
@@ -71,6 +77,13 @@ class taoQtiTest_actions_Runner extends tao_actions_ServiceModule
 {
     use RunnerToolStates;
 
+    /**
+     * Controls whether launching a new delivery suspends other sessions by the same user.
+     *
+     * @var string
+     */
+    public const FEATURE_FLAG_PAUSE_CONCURRENT_SESSIONS = 'FEATURE_FLAG_PAUSE_CONCURRENT_SESSIONS';
```

Comment:
We do not use constants for the feature flag itself, we just call its value. It is not necessary. Also, a controller/action is not a place to it. 

- [ ] Please document this feature flag on the README.md instead. Example: https://github.com/oat-sa/tao-core#environment-variables